packages/server/src/api/controllers/row/index.ts

```diff
@@ -72,6 +72,11 @@ export const save = async (ctx: UserCtx<Row, Row>) => {
   const tableId = utils.getTableId(ctx)
   const body = ctx.request.body
 
+  // user metadata doesn't exist yet - don't allow creation
```

Comment:
Could we add some basic tests to make sure this behaviour does not get removed in the future?